benchmark/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # CMake info 
 #-------------------------------------------------------------------------------------------
 cmake_minimum_required(VERSION 3.1)
-project(EASTLBenchmarks)
+project(EASTLBenchmarks CXX)
```

Comment:
Is there a benefit to explicitly including the 'CXX'?  The documentation I've read says that the project enabled C/CXX by default.  https://cmake.org/cmake/help/v3.0/command/project.html